### PR TITLE
Readme suggestions

### PR DESCRIPTION
**Pull request by ofrankowska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-- Run `yarn dev` in **Snowdog_Components** directory to see components in Fractal environment.
-## Requirements
-  * Working and valid Magento 2.4 instance.
-  * Node >= 16.
+- Run `yarn dev` in `app/design/frontend/Snowdog/CHILD_THEME_NAME/Snowdog_Components` directory to see components in Fractal environment.
 
-## User guide
-- [Working with Alpaca Theme]() - Learn how to utilize Alpaca Theme.
-- [Alpaca Theme step by step setup]() - How to set up Alpaca from scratch, without **create-alpaca-theme**.
-- [Current Alpaca Release](https://github.com/SnowdogApps/magento2-alpaca-theme) - Visit Alpaca's Github page.
```
Comment (ofrankowska) [inline README.md:19]: the last point doesn't seem related to this section

File: README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
 # Create Alpaca Theme
 
-Quickly create Magento 2 Child Theme based on Snowdog Alpaca Theme.
+Quickly create Magento 2 Child Theme based on [Snowdog Alpaca Theme](https://github.com/SnowdogApps/magento2-alpaca-theme).
+
+## Requirements
```
Comment (ofrankowska) [inline README.md:5]: I think this should be before installation

File: README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-## Requirements
-  * Working and valid Magento 2.4 instance.
-  * Node >= 16.
+- Run `yarn dev` in `app/design/frontend/Snowdog/CHILD_THEME_NAME/Snowdog_Components` directory to see components in Fractal environment.
 
-## User guide
-- [Working with Alpaca Theme]() - Learn how to utilize Alpaca Theme.
-- [Alpaca Theme step by step setup]() - How to set up Alpaca from scratch, without **create-alpaca-theme**.
-- [Current Alpaca Release](https://github.com/SnowdogApps/magento2-alpaca-theme) - Visit Alpaca's Github page.
 ## What it does?
```
Comment (ofrankowska) [inline README.md:20]: this seems fitting after quick overview

**Review by patrykbura — APPROVED** [review]
